basic_test.go

```diff
@@ -25,6 +25,17 @@ func (EventA) String() string {
 	return "Oh, Hello"
 }
 
+func TestDefaultSubIsBuffered(t *testing.T) {
+	bus := NewBus()
+	s, err := bus.Subscribe(new(EventA))
+	if err != nil {
+		t.Fatal(err)
+	}
+	if cap(s.(*sub).ch) == 0 {
```

Comment:
Wait, no why is this 0, and the setting isn't applied in https://github.com/libp2p/go-eventbus/pull/13/files#diff-cfdd0e8ab37b87e3592a157a92d85fa6R145

Reply:
I don't understand. If cap is zero then channel is not buffered and we error out the test.

Reply:
Nevermind, I didn't have enough coffee today.